packages/fxa-react/jest.config.js

```diff
@@ -1,3 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
It feels a little funny having some config files root level and others inside a `config/` dir. Should they all just be top-level? 🤔

Reply:
This config and all others at the root are for fxa-react exclusively. The configs inside `/configs` are meant to be shared.